packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -429,6 +442,15 @@ const DailyItemTasks: GarboTask[] = [
     do: () => BurningLeaves.burnSpecialLeaves($item`day shortener`),
     spendsTurn: false,
   },
+  {
+    name: "Burning Leaves Best Unlimited Tradeable Item",
```

Comment:
i'm not 100% we should be doing this yet

Reply:
we don't trade anything else unlimited right now from iotms and i just don't want to burn all of someone's leaves if they're saving up for something. it also wouldn't automatically equalize the mall right away unless everything generated makes it way into the mall and not everyone has philter set up